src/actions/mod.rs

```diff
@@ -65,8 +65,13 @@ impl ActionHandler {
         }
     }
 
-    pub fn init<O: Output>(&self, out: O) {
-        self.build_current_project(BuildPriority::Cargo, out);
+    pub fn init<O: Output>(&self, init_options: Option<InitializationOptions>, out: O) {
+        let omit_init_build = init_options.and_then(|x| x.omit_init_build)
```

Comment:
Sort of. I would implement `Default` for `InitializationOptions` and take a `InitializationOptions` here, rather than an `Option`